### PR TITLE
Milestone 6 - deployment & documentation

### PR DESCRIPTION
**Pull request by SarvarKh**
**The Next Web clone Milestone 6 - deployment & documentation**| 
In this milestone we:
- added tablet and desktop version view;
- rectified linter errors;
- corrected remaining issues while re-sizing the browser-screen;

### CODE REVIEW

**Comment by uryelah** [comment]
Hi  **Gerald Enekwechi & Sarvar Khalimov**! 👋 

I'm marking this Code review request as **INVALID** because this PR(Pull Request) currently does not meet the standards of a good descriptive title and message.

Please make sure to review the _3 basic rules to follow in Microverse projects_ step at [the professional repo guide from Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) and edit your PR title and message in a way that they are meaningful and descriptive of the code you are submitting.

**Example of a good PR message:**
69548761-3a8d3600-0f98-11ea-9f19-14da361c12e2

----

### Please _re-submit_ your project for review after updating the PR message and title accordingly.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,622 @@
...
+        <div class="nav-bottom-2-icon nav-icon">
+          <i class="fas fa-bars"></i>
+        </div>
+      </div>
+    </div>
+  </header>
+  <section>
+    <div class="main">
+
+      <!-- Heading -->
```
Comment (uche-inyama) [inline index.html]: Use the main tag to wrap elements within the main section [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi team :wave: 

Your design looks good, however, it would be nice to:

- [ ]  Increase the grid size of **Latest deals** and **Latest news** section to 4 grids and 3 grids  @media >=950px and @media >= 800 px respectively.

- [ ] Ensure to use the standard widths while implementing your media queries. These widths are: 

                     // small devices
                    @media (min-width: 576px) { ... }

                      // Medium devices (tablets, 768px and up)
                     @media (min-width: 768px) { ... }

                      // Large devices (desktops, 992px and up)
                      @media (min-width: 992px) { ... }

                       // Extra large devices (large desktops, 1200px and up)
                       @media (min-width: 1200px) { ... }

That's it, make the changes and submit another form for a code review

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @SarvarKh and @Oxford-G :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**